rtos/rtx/TARGET_CORTEX_M/RTX_CM_lib.h

```diff
@@ -361,6 +361,8 @@ extern void pre_main (void);
 static uint32_t thread_stack_main[DEFAULT_STACK_SIZE / sizeof(uint32_t)];
 #elif defined(TARGET_XDOT_L151CC)
 static uint32_t thread_stack_main[DEFAULT_STACK_SIZE * 6 / sizeof(uint32_t)];
+#elif defined(TARGET_REALTEK_RTL8195AM)
```

Comment:
@bulislaw the whole thread_stack_main config can be moved to each target's mbed_rtx.h if 4K is not a hard limit.